docs/source/contents/4_draft_mapping.rst

```diff
@@ -2331,12 +2464,14 @@ Temporal subjects
 Use Case
 ^^^^^^^^
 
-:code:`temporal` :code:`subject`\ s share information about a time period using text or a date (:code:`edtf`). None of our existing :code:`temporal` :code:`subject`\ s include URIs.These values are prominent in Volunteer Voices and the Pi Beta Phi to Arrowmont collections.
+:code:`subject/temporal` values share information about a time period using text or a date (:code:`edtf`). None of our existing :code:`subject/temporal`
+values include URIs.These values are prominent in Volunteer Voices and the Pi Beta Phi to Arrowmont collections. While not from established controlled
```

Comment:
"... include URIs. These values ..." missing a space.

Reply:
Thank you! I've addressed this.